Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.6-alpine
+ENV PYTHONUNBUFFERED 1 \
+    PY_ENV development
+
+WORKDIR /app
+
+ADD requirements.txt /app/
+
+RUN apk --no-cache add --virtual build-dependencies build-base \
```

Comment:
Ah, I missed this before - to help with caching we could put the `apk add` before the `ADD requirements.txt /app/` and only run `pip install -r ...` here, that way ideally we are caching anything that may change